pkg/graph/schema.graphql

```diff
@@ -86,6 +86,48 @@ type CedarAuthorityToOperate {
   xlcPhase: String
 }
 
+"""
+CedarSoftwareProductItem represents an individual software product; this information is returned from the CEDAR Core API
+as a part of the CedarSoftwareProducts object
+"""
+type CedarSoftwareProductItem {
+  apiGatewayUse: Boolean
+  elaPurchase: String
+  elaVendorId: String
+  providesAiCapability: Boolean
+  refstr: String
+  softwareCatagoryConnectionGuid: String
+  softwareVendorConnectionGuid: String
+  softwareCost: String
+  softwareElaOrganization: String
+  softwareName: String
+  systemSoftwareConnectionGuid: String
+  technopediaCategory: String
+  technopediaID: String
+  vendorName: String
+}
+
+"""
+CedarSoftwareProducts represents the response from the /softwareProducts endpoint from the CEDAR Core API.
+Right now, this does not tie in with any other types defined here, and is a root node until that changes. - NJD is this right about root node?
+"""
+type CedarSoftwareProducts {
+  aiSolnCatg: [String]!
+	aiSolnCatgOther: String
```

Comment:
Is this formatting intentional?

Reply:
No - it looks ok in my VSCode for some reason. I'll make sure this is fixed.

Reply:
nice, thanks